terragrunt/aws/dynamodb/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "url_shortener_table_name" {
```

Comment:
Minor nit - could you add a `description` to each of these outputs.

Reply:
Done!